src/librustdoc/visit.rs

```diff
@@ -1,5 +1,11 @@
 use crate::clean::*;
 
+/// allows a type to traverse the cleaned ast of a crate.
```

Comment:
Missing capitalized letter.

Reply:
see https://github.com/rust-lang/rust/pull/130537#discussion_r1766212397

Reply:
Please do it. Comments should be capitalized.

Reply:
Is that an official policy? checking the rustc dev guide, the std dev guide, and the [api docs conventions rfc](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text), there's no comment on capitalization preference.

currently deferring to previous approval, and the golden rule of code formatting (match the existing formatting of the file you're editing).

Reply:
And you'll see across the files that both styles are there. So please use the one with capital and even send a PR to fix the rest if you want.

Reply:
the current file (`visit.rs`) only uses lowercase.  this is mainly because a lot of its doc comments are not full sentences.

in our postmodern english, capitalizing a sentence fragment is arguably worse than not capitalizing a full sentence.  to make these docs fully idiomatic, i would need to rephrase all the sentence fragments into full sentences, which isn't trivial.

really, the comments on the trait methods could be more helpful, but i don't have the expertise to fully explain their purpose.

perfect is the enemy of good.

Reply:
You realize that doing what I'm asking would have allowed this PR to have been merged a lot earlier, right?

Reply:
It probably could've been merged faster if you were less insistent on enforcing non-existent style guide rules, but if you really care that much, i guess i can capitalize it.

Reply:
honestly there probably should be at least a suggestion to use proper capitalization, expecially for official documentation.

enforcing an unwritten rule still feels pretty bad.

Reply:
I'm used to have sentences start with a capital letter and end with a dot. The only cases where I don't enforce it is on big PRs where the comments are finally just a small part of and I don't want to slow the merge.